lua/graph.lua

```diff
@@ -1482,20 +1482,20 @@ function filter_tags_generic(kv)
   end
 
   lane_count = numeric_prefix(kv["lanes"],false)
-  if lane_count and lane_count > 10 then
-    lane_count = 10
+  if lane_count and lane_count > 15 then
```

Comment:
before it was if we have more than 10 we cap at 10, now its if we have more than 15 we have none?

Reply:
![image](https://user-images.githubusercontent.com/2676277/58715226-64bcbd00-8394-11e9-83ed-bf1fd2bc3b94.png)